<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="PersonalProject1/tests" tests="1" failures="1" skipped="0" hostname="HP-Gaganrana" time="2.342" timestamp="Wed Feb 02 16:53:44 IST 2022">
	<properties>
		<property name="testEnvironment" value=""/>
		<property name="pluginOutputLevel" value="WARNING"/>
		<property name="projectPath" value="C:/Users/gaggan.rana/Provar/PersonalWS1/PersonalProject1"/>
		<property name="salesforceMetadataCache" value="../.provarCaches"/>
		<property name="resultsPathDisposition" value="Increment"/>
		<property name="testOutputLevel" value="BASIC"/>
		<property name="webBrowserType" value="Chrome"/>
		<property name="resultsPath" value="C:/Users/gaggan.rana/Provar/PersonalWS1/PersonalProject1/ANT/Results"/>
		<property name="webBrowserConfiguration" value="Full Screen"/>
		<property name="projectCachePath" value="../.provarCaches"/>
	</properties>
	<testcase classname="tests/C:\Users\gaggan.rana\Provar\PersonalWS1\PersonalProject1" name="Test Case 2.testcase" time="2.341">
		<failure message="Execution failed"><![CDATA[Connection exception while opening Partner connection. Connection name: PersonalOrg, environment: , authEndpoint: https://innovation-efficiency-159-dev-ed.cs114.my.salesforce.com/services/Soap/u/53.0, userName: gagganscratchorg1@provartesting.com, password: [supplied], securityToken: [empty]]]></failure>
	</testcase>
</testsuite>
